package.json

```diff
@@ -8,7 +7,8 @@
         "watch": "lerna run watch",
         "start": "lerna run start",
         "test": "lerna run test --",
-        "publish:latest": "lerna publish --registry=https://registry.npmjs.org/ --exact --no-git-tag-version --no-push",
+        "publish:latest:ci": "lerna run publish:latest:ci",
+        "publish:latest:manual": "lerna publish --registry=https://registry.npmjs.org/ --exact --no-git-tag-version --no-push --yes",
```

Comment:
I was not sure, but I thought it could still be useful for now to be able to manually publish, so I kept this script but renamed it to ":manual"